src/main/kotlin/com/hibob/kotlinBascis/StoreService.kt

```diff
@@ -0,0 +1,78 @@
+package com.hibob.kotlinBascis
+import org.springframework.stereotype.Component
+import java.time.LocalDate
+
+@Component
+class StoreService {
+
+    fun pay(cart: List<Cart>, payment: Payment): Map<String, Check> {
+        val ans = mutableMapOf<String, Check>()
+        cart.forEach{myCart->
+            val check=checkout(myCart, payment)
+            val clientId:String=myCart.clientId
+            ans[clientId]=check
+        }
+       return ans
+    }
+}
+
+
+fun checkCardType(type: CreditCardType):Boolean{
```

Comment:
nice function but maybe change the name to something more indicate, for example isCardTypeSupported.